src/FileService.Mappers/RequestMappers/ImageRequestMapper.cs

```diff
@@ -33,15 +34,19 @@ public DbImage Map(ImageRequest imageRequest, ImageType imageType, Guid? parentI
 
             if (imageType == ImageType.Full)
             {
+                using Image image = Image.Load(imageRequest.Content);
+                isBigImage = image.Width > 150 && image.Height > 150;
```

Comment:
What if image will have long size only on Height or Weight? like Height = 1000 and Weight =130
Is it ok image or not?

Reply:
no, we need only when both parametrs fulfil the conditions